spec/support/shared_entity_specs.rb

```diff
@@ -133,3 +133,55 @@ def verify_signature(pubkey, signature, signed_string)
     end
   end
 end
+
+shared_examples "a JSON Entity" do
+  it "#to_json output matches JSON schema" do
+    json = described_class.new(data).to_json
+    expect(json.to_json).to match_json_schema(:entity_schema)
+  end
+
+  include_examples "common Entity JSON expectations" do
```

Comment:
> I would add that to the `shared_examples "a JSON Entity"`

I thought about just merging the `shared_examples` instead of including it . If we test this in `"a JSON Entity"` for every entity, we don't need to test `"common Entity JSON expectations"` for testing for only `Entity` and `Relayable`.